.github/workflows/docker.yml

```diff
@@ -85,6 +91,13 @@ jobs:
           build-args: |
             SERVER_IMAGE=temporaliotest/server:${{ steps.build_args.outputs.image_tag }}
 
+      - name: Run Trivy vulnerability scanner on Admin Tools image
```

Comment:
are we going to see triple warnings for things that are present in all three images?

Reply:
Yes, as I understand, all three scan results will be available.